scripts/run_backend_tests.py

```diff
@@ -659,6 +695,25 @@ def check_coverage(
     process = subprocess.run(
         cmd, capture_output=True, encoding='utf-8', env=env,
         check=False)
+
+    not_covered_lines = []
+    is_first_line = True
+    output_lines = process.stdout.split('\n')
+
+    for index, line in enumerate(output_lines):
+        if line and (' 100%' not in line and '-----' not in line
+         and 'Name' not in line):
+            if is_first_line and index > 0:
```

Comment:
Not sure what `index` refers to here. Maybe give it a more specific name?

Reply:
@seanlip i included correct name as suggested by you in the updated code. Thanks.